src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/Menu.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<!--
+    This Source Code Form is subject to the terms of the MIT License.
+    If a copy of the MIT was not distributed with this file, You can obtain one at https://opensource.org/licenses/MIT.
+    Copyright (C) Leszek Pomianowski and WPF UI Contributors.
+    All Rights Reserved.
+-->
+
+<ResourceDictionary
+    x:Class="Wpf.Ui.Styles.Controls.Menu"
```

Comment:
This style has a counter part in Menu.xaml.cs, I think that needs to be brought here as well. See DataGridHeaderBorder class for reference, that may help in bringing the counter part here.

Reply:
Or you can simpley add Menu.xaml.cs along with the .xaml style here.

Reply:
Added Menu.xaml.cs file.